packages/js/jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  coveragePathIgnorePatterns: ['/node_modules/', '/lib/'],
+  coverageReporters: ['text-summary', 'lcov'],
+  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  setupFilesAfterEnv: ['./jest.setup.js'],
+  testEnvironment: 'jsdom',
```

Comment:
maybe no need for js/js SDKs should be default node

Reply:
removed this line. (`testEnvironment` defaults to `node`)